internal/db/schema/migrations/postgres/dev/86_iam.up.sql

```diff
@@ -0,0 +1,60 @@
+begin;
+
+-- add the account_info_auth_method_id which determines which auth_method is
+-- designated as for "account info" in the user's scope.  
+alter table iam_scope
+add column account_info_auth_method_id wt_public_id -- allowed to be null and is mutable of course.
+references auth_method(public_id)
```

Comment:
I think the foreign key should include the `scope_id` to ensure the scope owns the auth method.

Reply:
ty!